src/components/List.js

```diff
@@ -136,8 +137,8 @@ const List = ({ token }) => {
   };
 
   return (
-    <div>
-      <h1>List</h1>
+    <div className="list-container">
+      <Header />
```

Comment:
I added the Header component to the List and Add Item components and not the App component, because the Welcome page will have its own styling and won't include the header or navigation elements.